.github/workflows/mutation-testing.yml

```
@@ -0,0 +1,181 @@
+        required: false
+        default: '80'
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  javascript-mutation:
+    name: JavaScript Mutation Testing
+    runs-on: ubuntu-latest
+    if: hashFiles('package.json') != ''
```

actionlint (1.7.8)

calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)